src/components/cylc/Info.vue

```diff
@@ -75,9 +71,12 @@ along with this program.  If not, see <http://www.gnu.org/licenses/>.
               </a>
             </dd>
             <v-divider />
+            <dt>Description</dt>
+            <dd><Markdown :markdown="taskMetadata.description || ''"/></dd>
+            <v-divider />
             <template v-for="(value, key) in customMetadata" :key="key">
-              <dt>{{ key }}</dt>
-              <dd><span class="markup">{{ value }}</span></dd>
+              <dt><Markdown :markdown="'## ' + key"/></dt>
```

Comment:
No need to handle the key as markdown (it's a Cylc config item), only the value. If we cant to change its appearance of this header, we can use CSS.

```suggestion
              <dt>{{ key }}></dt>
```